arduino/src/main.cpp

```diff
@@ -20,40 +24,92 @@ void setup(){
     while(WiFi.status() != WL_CONNECTED){
         Serial.print(".");
         delay(100);
+        
     }
-
-    
+    pinMode(LED,OUTPUT);
+    pinMode(BUTTON,INPUT_PULLUP);
 }
-
-void loop() {
-    // Odczytaj stan GPIO
-   int gpioState = 10 ;
+int send_status_request(int buttonState) 
+{
+    
     DynamicJsonDocument jsonDoc(200);
-    jsonDoc["gpioState"] = gpioState;
+    jsonDoc["place"] =  MIEJSCE;
+    if(buttonState==LOW)
+    {
+        jsonDoc["state"] = "hanged";
+        delay(30000);
+    }
+    else
+    {
+        jsonDoc["state"]="empty";
+        delay(30000);

```

Comment:
Po co te delaye?

Reply:
żeby informacja o stanie była przesyłana w odstępach 30 sekundowych

Reply:
da się usawić  żeby  serwer sprawdzał informacje od płytki co 30 a nie żeby płytka je wysyłała w takich odstępach? Jeżeli ustawimy delay w kodzie płytki to również diody zapalają się i gaszą z podanym opóźnieniem.

Reply:
Z samej definicji serwera wynika to, że to właśnie on jest odbiorcą wszelkich wiadomości. Jedyną rzeczą do zrobienia tutaj jest poprawienie kodu samej płytki.